test/uk/gov/hmrc/merchandiseinbaggage/stubs/MibBackendStub.scala

```diff
@@ -19,37 +19,38 @@ package uk.gov.hmrc.merchandiseinbaggage.stubs
 import com.github.tomakehurst.wiremock.WireMockServer
 import com.github.tomakehurst.wiremock.client.WireMock._
 import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import com.google.inject.Inject
 import play.api.libs.json.Json
 import play.api.libs.json.Json.toJson
 import uk.gov.hmrc.merchandiseinbaggage.CoreTestData
-import uk.gov.hmrc.merchandiseinbaggage.config.MibConfiguration
+import uk.gov.hmrc.merchandiseinbaggage.config.AppConfig
 import uk.gov.hmrc.merchandiseinbaggage.model.api.DeclarationType.Export
 import uk.gov.hmrc.merchandiseinbaggage.model.api._
 import uk.gov.hmrc.merchandiseinbaggage.model.api.calculation._
 
-object MibBackendStub extends MibConfiguration with CoreTestData {
+class MibBackendStub @Inject() (config: AppConfig) extends CoreTestData {
 
   val stubbedDeclarationId = DeclarationId("test-mib-be-id")
 
   def givenDeclarationIsPersistedInBackend(declaration: Declaration)(implicit server: WireMockServer): StubMapping =
     server
       .stubFor(
-        post(urlPathEqualTo(s"$declarationsUrl"))
+        post(urlPathEqualTo(s"${config.mibDeclarationsUrl}"))
           .withRequestBody(equalToJson(toJson(declaration).toString, true, false))
           .willReturn(aResponse().withStatus(201).withBody(Json.toJson(declaration.declarationId).toString))
       )
 
   def givenDeclarationIsPersistedInBackend(implicit server: WireMockServer): StubMapping =
     server
       .stubFor(
-        post(urlPathEqualTo(s"$declarationsUrl"))
+        post(urlPathEqualTo(s"${config.mibDeclarationsUrl}"))
```

Comment:
This can just be `post(urlPathEqualTo("/declare-commercial-goods/declarations"))`